src/framework/shortcuts/internal/shortcutsregister.cpp

```diff
@@ -80,12 +80,12 @@ void ShortcutsRegister::load()
     }
 }
 
-void ShortcutsRegister::mergeSortcuts(ShortcutList& shortcuts, const ShortcutList& defaultShortcuts) const
+void ShortcutsRegister::mergeShortcuts(ShortcutList& shortcuts, const ShortcutList& defaultShortcuts) const
 {
     ShortcutList needadd;
     for (const Shortcut& sh : defaultShortcuts) {
         auto it = std::find_if(shortcuts.begin(), shortcuts.end(), [sh](const Shortcut& i) {
-            return i.action == sh.action;
+            return i.action == sh.action && i.sequence == sh.sequence;
```

Comment:
actually i'm not sure what the right behaviour is here - is the intention that you can *replace* shortcuts rather than merge/add them?

Reply:
Reset to factory settings doesn't actually work, you have to delete the local file (for me it was  C:/Users/dylan.nicholson/AppData/Local/MuseScore/MuseScore4Development/shortcuts.xml)
But I think requiring existing MU4 users (remembering it's still pre-alpha!) and restoring the existing logic is the only correct solution, or if you removed any of the default shortcut mappings, they'd just come back again next time you started up MuseScore.